### PR TITLE
Code Review for Initial setup with components at BookStore project.

### PR DESCRIPTION
**Pull request by luigirazum**
# Hello Dear Code Reviewers Team 👋,
### The following requirements 📌 were fulfilled this time:
- 📌 requirement X
    - ✅ Done.
- 📌 Initialise React app.
    - ✅ Done. Using create-react-app and removing the boilerplate code.
- 📌 The building blocks of your app should be set as re-usable components. Create a directory for your components.
    - ✅ Done. A `components` folder is inside the `src` folder.
- 📌 Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
    - ✅ Done. Default route renders the Book List, another route for Categories and the * that renders an error page.
    - 📌 Books - the default view:
        - 📌 Should display the list of books (empty at this point but it should be ready for the data) with the Remove button.
            - ✅ Done. The list of `Book`'s is generated, using a default array for now.
        - 📌 Create a component called Book for displaying a single book (receive a title and an author as prop).
            - ✅ Done. A `Book` component was created.
        - 📌 Should display a form for adding a book (no functionality yet).
            - ✅ Done. It's just where it should be.
        - 📌 Create a separate component for this form (with inputs for a title and an author).
            - ✅ Done. It's right at the bottom.
    - 📌 Categories:
        - 📌 Should display a button "Check status" only.
            - ✅ Done. It's there.
- 📌 Styling is not required at this point.
    - ✅ Done. No styles were applied.
### Luis

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const Book = (props) => {
+  const { book: { title, author } } = props;
+
+  return (
+    <div>
+      <h2>{title}</h2>
+      <h4>{author}</h4>
+    </div>
+  );
+};
```
Comment (Tadesse-Alemayehu) [inline src/components/Book.js:13]: - [ ] Impressive job @luigirazum, The one thing missing is the `remove button` alongside the book title and author. the book component supposes to have a button for removing books, I see you have the button but we need to return that button from the `book` component.

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @luigirazum 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by luigirazum** [comment]
Thanks for the feedback.
Changes applied, Move the Button component from BooksList to the Book.

I'm asking for the review.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @luigirazum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**